scripts/helpers.js

```diff
@@ -207,6 +207,25 @@ hexo.extend.helper.register('docs_root_path', function (pages, version) {
 });
 
 /**
+ * Retrieve the page with `installation: true` or the second page.
+ */
+hexo.extend.helper.register('docs_getting_started_path', function (pages, version) {
+  var installPages = docs_version_filter(pages, version).find({
+    parent_section: 'docs',
+    installation: true
+  });
+
+  if (installPages.data.length) {
+    return '/' + installPages.data[0].path.replace('index.html', '');
```

Comment:
why not do this below? specific to `installPages`? sorry, haven't looked at this in a while; I'm a bit lost.